build/GridItem.js

```diff
@@ -70,7 +70,7 @@ var GridItem = React.createClass({
     handle: React.PropTypes.string
   },
 
-  getDefaultProps: function () {
+  getDefaultProps: function getDefaultProps() {
```

Comment:
Gotcha. I had an old version in npm cache. Could you rebase to master so we can squash the diffs?